.github/workflows/snyk.yml

```diff
@@ -0,0 +1,20 @@
+name: Snyk
```

Comment:
Have you tested this @lukeindykiewicz ? I think it won't work without adding the `sbt-dependency-graph` plugin to the project.

To avoid adding one more plugin, you can forego the default Snyk action and instead do it like here: https://github.com/snowplow/scala-maxmind-iplookups/blob/master/.github/workflows/snyk.yml.

Reply:
Thanks @dilyand for spotting this!

Reply:
fixed in 314c3de